README.md

```diff
@@ -110,8 +110,12 @@ Then, in your favorite form library, simply pass in the `changeset` in place of
 ```hbs
 {{! dummy-form/template.hbs}}
 <form>
-  <Input @value={{changeset.firstName}} />
-  <Input @value={{changeset.lastName}} />
+  <input
+    value={{model.firstName}}
+    oninput={{action (changeset-set this.changeset "model.lastName") value="target.value"}}>
```

Comment:
@josemarluedke Do these README updates look ok to you?  ie. the `model` as the `value` as noted https://github.com/poteto/ember-changeset/issues/398#issuecomment-573366958

Reply:
I don't think this change is correct. I believe forms should always reflect the in-flight model. This way, we can revert changes and have forms reflect it without any issues. Also, If you have a form in a tab layout for example, and you hide half of the form then go back to that, the form would show the incorrect data, the user would think they have lost their changes.

Reply:
reverted.  Thanks a ton for the insight.